Adapter_oM/enums/PushType.cs

```diff
@@ -40,6 +40,8 @@ public enum PushType //add error message if not picked up --> compliance check f
         CreateNonExisting,
         [Description("Uses only the Update CRUD method to update the objects in the external software. All other objects in the model are left untouched.")]
         UpdateOnly,
+        [Description("Attempt to Update the objects if possible, otherwise Create them.")] // Deletion is not included in this type.
+        UpdateOrCreate,
```

Comment:
```suggestion
        UpdateOrCreateOnly,
```
Suggested addition of "Only" to clarify naming convention alongside the other PushTypes